maps/cogmap2.dmm

```diff
@@ -77490,10 +77490,13 @@
 	},
 /area/research_outpost)
 "fzu" = (
-/obj/machinery/computer/research_shuttle/embedded/south{
-	name = "Research Outpost Shuttle Control"
-	},
 /obj/stool/chair/comfy/shuttle/green,
+/obj/machinery/computer/transit_shuttle/research{
+	icon_state = "shuttle-embed";
+	pixel_y = -25;
+	ejectdir = 2;
+	embed = 1
+	},
```

Comment:
Shouldn't the icon state and pixel shift be set in code and not in the map file?